style.css

```diff
@@ -0,0 +1,9 @@
+body{
+    background-color:lightgray 
+}
+.div1 {
+    width: 300px;
+    height: 100px;
+    border: 8px solid blue;
+  }
+}
```

Comment:
You have an extra closing curly brace here. Your CSS file isn't valid and won't show up with this. Fortunately, eslint will make it easier to detect this in projects moving forward.